dev/source/docs/style-guide.rst

```diff
@@ -4,8 +4,7 @@
 ArduPilot Style Guide
 =====================
 
-In order to improve the readability of the ArduPilot code and help new
-users pick up the code quickly please use the following styles.
+ArduPilot Style Guide is to improve the readability, reliability and error proof reading of code for new users to pick up the code quickly, please use the following styles.
```

Comment:
English grammar checker says this is fine except for adding a comma:
-after "code"     after "quickly"

also I would add " Code submitted may be rejected if these styles are not followed" at the end